### PR TITLE
Add some virtual_switches request implementation

### PR DESCRIPTION
**Pull request by CallisteH**
added method virtual_switch
added method virtual_switches

made some code reformatting to make it compliant with rubocop

### CODE REVIEW

File: lib/ibm_power_hmc/connection.rb [hunk 1]
```diff
@@ -213,7 +213,7 @@ def lpar_profiles(lpar_uuid)
     # @param params [Hash] Job parameters.
     # @param sync [Boolean] Start the job and wait for its completion.
     # @return [IbmPowerHmc::HmcJob] The HMC job.
-    def poweron_lpar(lpar_uuid, params = {}, sync = true)
+    def poweron_lpar(lpar_uuid, params = {}, sync: true)
```
Comment (dberg1) [inline lib/ibm_power_hmc/connection.rb:216]: Would this require changes to callers in manageiq-providers-ibm_power_hmc?

Comment (CallisteH) [reply lib/ibm_power_hmc/connection.rb:216]: @dberg1 That's a good question, I haven't tested this feature yet. I will try to work on it today to test if it works.

**Comment by CallisteH** [comment]
Thanks @dberg1, I just did it in the last commit for this pull request.

**Comment by jaywcarman** [comment]
Looking good @CallisteH - when you're ready to merge just remove `[WIP]` from the title.

**Pull request merged**